src/components/musette/musette.stories.js

```diff
@@ -0,0 +1,136 @@
+import { html } from ".storybook/helpers";
+import { withKnobs } from "@storybook/addon-knobs";
+import lesMusettes from "./musette.js";
+import { useEffect } from "@storybook/client-api";
+
+export default { title: "Carousel/Musette", decorators: [withKnobs] };
+
+const Musette = () => {
+	useEffect(() => { lesMusettes(); });
+	return html`
+	<div class="la-musette la-musette-no-scrollbar js-la-musette">
+		<div class="musette-fold">
+			<a href="http://www.metmuseum.org/art/collection/search/771396">
+				<img src="https://collectionapi.metmuseum.org/api/collection/v1/iiif/503578/1358034/main-image">
```

Comment:
probably better if you can change the `src` for all of these to one of our storybook image helpers, so we're not depending on non-local assets in test environments for snapshots. not a blocker but its preferred.